proto/viam/app/mltraining/ml_training.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package viam.app.mltraining.v1;
+
+import "app/data/v1/data.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/mltrainining/v1";
+
+service MLTrainingService {
+  rpc SubmitTrainingJob(SubmitTrainingJobRequest) returns (SubmitTrainingJobResponse);
+  rpc GetTrainingJob(GetTrainingJobRequest) returns (GetTrainingJobResponse);
+}
+
+enum ModelType {
+  MODEL_TYPE_UNSPECIFIED = 0;
+  MODEL_TYPE_MULTI_CLASS_CLASSIFICATION = 1;
+  MODEL_TYPE_MULTI_LABEL_CLASSIFICATION = 2;
+}
+
+message SubmitTrainingJobRequest {
+  viam.app.data.v1.Filter filter = 1;
+  string organization_id = 2;
+  string model_name = 3;
+  string model_version = 4;
+  ModelType model_type = 5;
+  repeated string training_labels = 6;
+}
+
+message SubmitTrainingJobResponse {
+  string id = 1;
+}
+
+message GetTrainingJobRequest {
+  string id = 1;
+}
+
+message GetTrainingJobResponse {
+  string id = 1;
```

Comment:
nit/q: Do we need id in both the request and response?

Reply:
So interestingly enough this is not a settled question and there's a lot of room for debate. Relevant [SO](https://stackoverflow.com/questions/11299354/should-a-restful-get-response-return-a-resources-id) (REST vs rpc, but this is a GET method for a resource so seems appropriate)

I personally lean towards including it as a general rule, because then the response can be referenced/used other than in the immediate context of when making the request. So for example if you have this response and want to refresh that, you can do that, even if you didn't save the id when you made the initial request.

We obviously don't do this in most places, though. If you think that inconsistency makes it confusing happy to remove too, I don't feel strongly

Reply:
Fine to keep, makes sense to me in the context of refreshing